src/main/java/org/openpnp/gui/MachineControlsPanel.java

```diff
@@ -419,8 +419,8 @@ public void mouseClicked(MouseEvent e) {
                     }
                     else if (e.getOldValue() != null && e.getNewValue() == null) {
                         for (int i = 0; i < comboBoxHeadMountable.getItemCount(); i++) {
-                            NozzleItem item = (NozzleItem) comboBoxHeadMountable.getItemAt(i);
-                            if (item.getNozzle() == e.getOldValue()) {
+                            HeadMountableItem item = (HeadMountableItem) comboBoxHeadMountable.getItemAt(i);
```

Comment:
This is just a bugfix. I think you added Actuators to the dropdown. Now those will create a cast exception.